### PR TITLE
fluent-bit: Allows configuration of Logstash_DateFormat for elasticse…

### PR DESCRIPTION
**Pull request by pms1969**
**What this PR does / why we need it**:

Allows for the configuration of elasticsearch Logstash_DateFormat, for configuring the ES index used by FluentBit.

see: https://fluentbit.io/documentation/0.13/output/elasticsearch.html

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE.

**Special notes for your reviewer**:
Included inside the Logstash_Prefix check.

### CODE REVIEW

File: stable/fluent-bit/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 0.9.0
+version: 0.9.2
```
Comment (davidkarlsen) [inline stable/fluent-bit/Chart.yaml:2]: where did 0.9.1 go?

**Comment by davidkarlsen** [comment]
@pms1969 can you sign off your commit please?
/ok-to-test

File: stable/fluent-bit/Chart.yaml (hunk 1 above)
Comment (pms1969) [reply stable/fluent-bit/Chart.yaml:2]: sorry, went to another PR that I submitted about 10 minutes before this one.  seems to have disappeared tho. :/

**Comment by pms1969** [comment]
@davidkarlsen how do I do that? an `/approve` comment?

File: stable/fluent-bit/Chart.yaml (hunk 1 above)
Comment (pms1969) [reply stable/fluent-bit/Chart.yaml:2]: found it: https://github.com/helm/charts/pull/7386

**Comment by unguiculus** [comment]
@pms1969 See https://github.com/helm/charts/pull/7393/checks?check_run_id=14307364

**Comment by pms1969** [comment]
ok, thanks.  in the middle of something right now, but will do so as soon as I'm over that.

**Comment by MaxDiOrio** [comment]
Can we revisit this one?  It's a pretty important change.  By default the indices created for each day create unnecessary bloat in Elastic since they're only a few meg to hundreds of megs in size.  By being able to set the date format to %Y.%m by default it does three things:

1) Allows larger shards for the whole month, closer to the 30GB recommended by Elastic.
2) Prevents creation with year, month,day and me having to go in, change the config map, reload the containers, then delete the indices in Elastic.
3) Allows upgrading the Helm Chart normally.  Each time the Helm chart gets updated, I have to redo this whole cycle in step 2, as it reverts my date format change.

Thanks.

**Pull request closed without merging**